pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -426,7 +430,16 @@ class Config11 {
             buildArgs : [
                     openj9 : '--ssh --disable-adopt-branch-safety -r git@github.ibm.com:runtimes/openj9-openjdk-jdk11 -b ibm_sdk'
             ],
-            test                : 'default',
+            test                : [
+                    weekly : [     
+                        'extended.functional.fips140_3_OpenJCEPlusFIPS',
+                        'sanity.jck.fips140_3_OpenJCEPlusFIPS',
+                        'extended.jck.fips140_3_OpenJCEPlusFIPS',
+                        'special.jck.fips140_3_OpenJCEPlusFIPS',
+                        'sanity.openjdk.fips140_3_OpenJCEPlusFIPS',
+                        'extended.openjdk.fips140_3_OpenJCEPlusFIPS'
+                    ]
+            ],
```

Comment:
If you overwrite the default, you must provide the full list, not just FIPS targets.

Reply:
https://github.com/ibmruntimes/ci-jenkins-pipelines/blob/ibm/pipelines/defaults.json#L44-L86